packages/cli/src/generators/templates/tsconfig.template.json

```diff
@@ -0,0 +1,18 @@
+{
+    "compilerOptions": {
+      "target": "ESNext",
+      "module": "NodeNext",
+      "moduleResolution": "NodeNext",
+      "resolveJsonModule": true,
+      "esModuleInterop": true,
+      "noUncheckedIndexedAccess": true,
+      "isolatedModules": true,
+      "noImplicitAny": false,
+      "skipLibCheck": true,
+      "outDir": "./dist",
+      "rootDir": "./src",
+      "strict": true
+    },
+    "include": ["./src/**/*.ts"],
+    "exclude": ["./**/*.test.ts"]
+  }
```

Comment:
Seems this is meant to replace the existing `tsconfig.json` in the same folder?

Reply:
Hmm, could you clarify this comment? This is meant to replace the `tsconfig.json` which lived in the `typescript` folder, since that whole folder has been deleted but we still need a template for `tsconfig.json`. Github doesn't show that very helpfully, it shows that it was [moved to the basic-auth-typescript example app here](https://github.com/zapier/zapier-platform/pull/1067/files#diff-7bd4839f12e83af3e80a2e79418f0b6086959d8ebfe943e91c136c0841a54726), but there shouldn't be another `tsconfig.json` in this folder 🤔